clab/export.go

```diff
@@ -0,0 +1,108 @@
+// Copyright 2020 Nokia
+// Licensed under the BSD 3-Clause License.
+// SPDX-License-Identifier: BSD-3-Clause
+
+package clab
+
+import (
+	"encoding/json"
+	"io"
+	"os"
+	"path/filepath"
+	"text/template"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/srl-labs/containerlab/types"
+)
+
+// GenerateExports generate various export files and writes it to a lab location
+func (c *CLab) GenerateExports() error {
+	topoDataFPath := filepath.Join(c.Dir.Lab, "topology-data.json")
+	f, err := os.Create(topoDataFPath)
+	if err != nil {
+		return err
+	}
```

Comment:
I followed an existing logic for `ansible-inventory.yml` in `inventory.go`. The file should be created in `deploy.go` so that it could be mounted to nodes, even while empty. I think I could remove `Create` from `export.go`.

Reply:
Removed from `export.go`.